build.sbt

```diff
@@ -170,36 +170,22 @@ lazy val metadataSettings = Def.settings(
 )
 
 lazy val dockerSettings = Def.settings(
-  dockerCommands := Seq(
-    Cmd("FROM", Option(System.getenv("DOCKER_BASE_IMAGE")).getOrElse("openjdk:8")),
-    Cmd("ARG", "DEBIAN_FRONTEND=noninteractive"),
-    ExecCmd("RUN", "apt-get", "update"),
-    ExecCmd("RUN", "apt-get", "install", "-y", "apt-transport-https", "firejail"),
-    ExecCmd(
-      "RUN",
-      "sh",
-      "-c",
-      """echo \"deb https://dl.bintray.com/sbt/debian /\" | tee -a /etc/apt/sources.list.d/sbt.list"""
-    ),
-    ExecCmd(
-      "RUN",
-      "apt-key",
-      "adv",
-      "--keyserver",
-      "hkp://keyserver.ubuntu.com:80",
-      "--recv",
-      "2EE0EA64E40A89B84B2DF73499E82A75642AC823"
-    ),
-    ExecCmd("RUN", "apt-get", "update"),
-    ExecCmd("RUN", "apt-get", "install", "-y", "sbt"),
-    Cmd("WORKDIR", "/opt/docker"),
-    Cmd("ADD", "opt", "/opt"),
-    ExecCmd("RUN", "chmod", "0755", "/opt/docker/bin/scala-steward"),
-    ExecCmd("ENTRYPOINT", "/opt/docker/bin/scala-steward"),
-    ExecCmd("CMD", "")
-  ),
+  dockerBaseImage := Option(System.getenv("DOCKER_BASE_IMAGE")).getOrElse("openjdk:8-jre-alpine"),
+  dockerCommands ++= {
+    val getSbtVersion = sbtVersion.value
+    val sbtTgz = s"sbt-$getSbtVersion.tgz"
+    val sbtUrl = s"https://github.com/sbt/sbt/releases/download/v$getSbtVersion/$sbtTgz"
+    Seq(
+      Cmd("USER", "root"),
```

Comment:
Could you switch back at the end to a non-privileged user to avoid running scala-steward as root?

Reply:
A PR for that would be welcome.

Reply:
Oh.. I was certain the setting `daemonUser` will be used in entrypoint. 
It doesn't work like the official description says:
- _daemonUser in Docker_
_The user to use when executing the application. Files below the install path also have their ownership set to this user._

There are two ways I can suggest to fix this. We can switch back to `daemonUser` in `dockerCommands` (just by adding another `Cmd(..)` to the end of the list) or modify `dockerEntrypoint` that will always start steward as `daemonUser`.
```scala
dockerEntrypoint := {
    val user = (Docker / daemonUser).value
    val runSteward = (dockerEntrypoint.value :+ "--disable-sandbox").toList
    "su" :: "-s" :: "/bin/bash" :: user :: "-c" :: runSteward
  }
```
Which way is more preferable @fthomas @mwz?

Reply:
Personally, I'd switch by running a new cmd at the end of docker commands.

Reply:
Running a new cmd seems to be the shorter solution. I would prefer that too, since it means less code to maintain.